src/components/ebay-notice/test/test.browser.js

```diff
@@ -0,0 +1,28 @@
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/browser');
+const renderer = require('../');
+
+describe('given the notice is in the default state', () => {
+    let widget;
+    let button;
+
+    beforeEach(() => {
+        widget = renderer.renderSync({ type: 'page', dismissible: true }).appendTo(document.body).getWidget();
+        button = document.querySelector('.page-notice__close');
+    });
+    afterEach(() => widget.destroy());
+
+    describe('when the dismiss button is clicked', () => {
```

Comment:
We should also test the programmatic access. You can see existing examples by searching for `programmatically`.